### ISSUE TITLE
omim text-search always returns empty results

### ISSUE THREAD

**Issue by stolpeo** [varfish-org/viguno#160]
**Describe the bug**
It appears that the search for OMIM terms can be performed by name, but it returns an empty list although a result is to be expected.

**To Reproduce**

```json
curl "localhost:8000/proxy/varfish/viguno/hpo/omims?omim_id=616145"
{
  "version": {
    "hpo": "2023-06-06",
    "viguno": "0.2.1"
  },
  "query": {
    "omim_id": "616145",
    "name": null,
    "match_": null,
    "max_results": 100,
    "hpo_terms": false
  },
  "result": [
    {
      "omim_id": "OMIM:616145",
      "name": "Catel-Manzke syndrome"
    }
  ]
}
```

```json
curl "localhost:8000/proxy/varfish/viguno/hpo/omims?name=catel"
{
  "version": {
    "hpo": "2023-06-06",
    "viguno": "0.2.1"
  },
  "query": {
    "omim_id": null,
    "name": "catel",
    "match_": null,
    "max_results": 100,
    "hpo_terms": false
  },
  "result": []
}
```

**Expected behavior**
The full-text search for omim names should return a result if available, in the example above the Catel-Manzke Syndrome.

**Comment by stolpeo** [comment]
Parameter `match=` can be used with options `prefix|suffix|exact`. Search is case-sensitive. Full-text HPO search on the other hand does not require a `match=` parameter and is case-insensitive.

```json
curl "localhost:8000/proxy/varfish/viguno/hpo/omims?name=Catel&match=prefix"
{
  "version": {
    "hpo": "2023-06-06",
    "viguno": "0.2.1"
  },
  "query": {
    "omim_id": null,
    "name": "Catel",
    "match_": "prefix",
    "max_results": 100,
    "hpo_terms": false
  },
  "result": [
    {
      "omim_id": "OMIM:616145",
      "name": "Catel-Manzke syndrome"
    }
  ]
}

```

**Comment by holtgrewe** [comment]
Will be fixed by #172 which allows case insensitive search:

![image](https://github.com/varfish-org/viguno/assets/521780/78c45a7e-18eb-4c8c-ae61-7ec8412292f1)
